server/clip_server/executors/clip_onnx.py

```diff
@@ -23,14 +23,20 @@ def __init__(
         device: Optional[str] = None,
         num_worker_preprocess: int = 4,
         minibatch_size: int = 32,
-        traversal_paths: str = '@r',
+        access_paths: str = '@r',
+        traversal_paths: Optional[str] = '@r',
         model_path: Optional[str] = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
 
         self._minibatch_size = minibatch_size
-        self._traversal_paths = traversal_paths
+        self._access_paths = access_paths
+        if traversal_paths is not None:
```

Comment:
Don't add the `traversal_paths` in the function signature. Let's use `kwargs` to pass `traversal_paths`